editor.planx.uk/src/pages/FlowEditor/index.tsx

```diff
@@ -99,14 +104,88 @@ export const LastEdited = () => {
       <Typography variant="body2" fontSize="small">
         {message}
       </Typography>
-      {hasFeatureFlag("UNDO") && formattedOps && (
-        <Typography component="ul" pl={2}>
-          {[...new Set(formattedOps)].map((op, i) => (
-            <Typography variant="body2" fontSize="small" component="li" key={i}>
-              {op}
-            </Typography>
-          ))}
-        </Typography>
+    </Box>
+  );
+};
+
+export const EditHistory = () => {
+  const [flowId, flow] = useStore((state) => [state.id, state.flow]);
+
+  const { data, loading, error } = useSubscription<{ operations: Operation[] }>(
```

Comment:
See above, `useSubscription()` also returns a `refetch()` function (if we want to use it!)